src/gui/update.rs

```diff
@@ -142,28 +142,24 @@ pub fn handle_message(ajour: &mut Ajour, message: Message) -> Result<Command<Mes
         }
         Message::Interaction(Interaction::Delete(id)) => {
             // Delete addon, and it's dependencies.
-            let addons = ajour.addons.clone();
-            if let Some(addon) = addons.iter().find(|a| a.id == id) {
+            if let Some(addon) = ajour.addons.iter().find(|a| a.id == id).cloned() {
```

Comment:
Instead of cloning the entire vec, just clone the `Option<&Addon>` into `Option<Addon>`